tests/test_gitlab.py

```diff
@@ -894,13 +910,15 @@ def test_initialization(self):
         self.assertEqual(client.rate_limit_reset_ts, None)
 
         client = GitLabClient("fdroid", "fdroiddata", "your-token", sleep_for_rate=True,
-                              min_rate_to_sleep=100, max_retries=10, sleep_time=100)
+                              min_rate_to_sleep=100, max_retries=10, sleep_time=100, is_oauth_token=True)
 
         self.assertEqual(client.sleep_for_rate, True)
         self.assertEqual(client.base_url, GITLAB_API_URL)
         self.assertEqual(client.min_rate_to_sleep, 100)
         self.assertEqual(client.sleep_time, 100)
         self.assertEqual(client.max_retries, 10)
+        self.assertEqual(client.is_oauth_token, True)
```

Comment:
As you similarly did for `TestGitLabBackend.test_initialization`, you should add assertions around line 910 to check that (i) `is_oauth_token` is False and (ii) the client headers are properly set.